src/pages/comparison/components/characteristics.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Rate } from './rate.component';
+
+import '../comparison.css';
+
+import { Pet } from '../interfaces/pet.interface';
+
+
+export const Characteristics = (props: {pets: Array<Pet>; display: string; rate: boolean }): React.ReactElement => {
+
+    const { pets, display, rate } = props;
+
+    const items = pets.map((el: Pet) => {
+        const property = Object.getOwnPropertyDescriptor(el, display);
+
+        const show = rate ? <Rate rate={property?.value} /> : <span>{property?.value}</span>;
+
+        const styles = {
+            color: 'black',
+            textDecoration: 'none'
```

Comment:
Not sure if it is a best practice to treat css styling like that, maybe it is better to use styles.css for styling

Reply:
Thanks, i will try to solve this issue